src/components/NumberInput/NumberInput.js

```diff
@@ -35,6 +35,9 @@ export const NumberInput = (props) => {
     autoComplete,
     maxLength,
     icon,
+    classOverrides,
```

Comment:
add these capabilities to `NumberInput` so we don't have to use custom CSS to overwrite. A follow up PR in CMS will use these properties for the `Age` question in EW.